Tools/runtests.ps1

```diff
@@ -23,4 +23,4 @@ if ($selectedVsVersion -eq '') {
 
 $testFiles = $tests | % { ".\BuildOutput\Release$selectedVsVersion\Tests\$_.dll" }
 
-& "$vstest" @args $testFiles
+& "$vstest" /settings:$(".\Build\default." + $selectedVsVersion + "Exp.testsettings") @args $testFiles
```

Comment:
Thanks, I didn't know about `PSScriptRoot` and it sounds like this would be a great improvement. I'll update the code to use it.